lib/resources.rb

```diff
@@ -1477,37 +1477,181 @@ def get_resource(key)
+                "use" => "usual",
+                "system" => "http://cerner.com/fhir/identifier/FINNBR",
+                "value" => "20001427"
+            }
+        ],
+        "status" => "in-progress",
+        "class" => "outpatient",
+        "patient" => {
+            "reference" => "Patient/1316020"
+        },
+        "participant" => [
+            {
+                "type" => [
+                    {
+                        "coding" => [
+                            {
+                                "system" => "http://hl7.org/fhir/v3/ParticipationType",
+                                "code" => "ATND",
+                                "display" => "attender"
+                            }
+                        ],
+                        "text" => "attender"
+                    }
+                ],
+                "individual" => {
+                    "reference" => "Practitioner/2386008",
+                    "display" => "Howser, Doogie"
+                }
+            }
+        ],
+        "period" => {
+            "start" => "2015-08-06T13:00:00.000Z"
+        },
+        "reason" => [
+            {
+                "text" => "Checkup"
+            }
+        ],
+        "location" => [
+            {
+                "location" => {
+                    "display" => "Outpatient Ctr"
+                }
+            }
+        ]
     }
 
     MAY2015_ENCOUNTER_BUNDLE ||= {
-      "resourceType" => "Bundle",
-      "id" => "40be3b40-6945-4db9-b993-2f67a44c77a6",
-      "type" => "searchset",
-      "base" => "https://fhir-open.sandboxcernerpowerchart.com/may2015/d075cf8b-3261-481d-97e5-ba6c48d3b41f",
-      "total" => 2,
-      "link" => [{"relation" => "self",
-                  "url" => "https://fhir-open.sandboxcernerpowerchart.com/may2015/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Encounter?patient=1316020"}],
-      "entry" => [{"resource" => MAY2015_ENCOUNTER},
-                  {"resource" =>
-                     {"resourceType" => "Encounter",
-                      "id" => "2777906",
-                      "status" => "in-progress",
-                      "patient" => {"reference" => "Patient/1316020"},
-                      "period" => {"start" => "2014-12-15T18:47:41.000Z"},
-                      "location" => [{"location" => {"display" => "Baseline East"},
-                                      "period" => {"start" => "2014-12-15T18:47:41.000Z"}}]}}]
+        "resourceType" => "Bundle",
+        "id" => "2cd77130-032f-4354-921e-d05141ca2184",
+        "type" => "searchset",
+        "base" => "https://fhir-open.sandboxcernerpowerchart.com/may2015/d075cf8b-3261-481d-97e5-ba6c48d3b41f",
+        "total" => 2,
+        "link" => [
+            {
+                "relation" => "self",
+                "url" => "https://fhir-open.sandboxcernerpowerchart.com/may2015/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Encounter?_id=3261906%2C2787906"
+            }
+        ],
+        "entry" => [
+            {
```

Comment:
We should still be able to use a reference to MAY2015_ENCOUNTER for the first resource to eliminate duplication.